install/install.go

```diff
@@ -120,11 +120,77 @@ func K8sInstaller(c *k8s.Client, o Options) error {
 		fmt.Print("KubeArmor Host Policy Manager Deployment already exists ...\n")
 	}
 
+	fmt.Print("KubeArmor Annotation Controller TLS certificates ...\n")
+	caCert, tlsCrt, tlsKey, err := GeneratePki(o.Namespace, deployments.AnnotationsControllerServiceName)
+	if err != nil {
+		fmt.Print("Could'nt generate TLS secret ...\n")
+		return err
+	}
+	if _, err := c.K8sClientset.CoreV1().Secrets(o.Namespace).Create(context.Background(), deployments.GetAnnotationsControllerTLSSecret(o.Namespace, caCert.String(), tlsCrt.String(), tlsKey.String()), metav1.CreateOptions{}); err != nil {
+		if !strings.Contains(err.Error(), "already exists") {
+			return err
+		}
+		fmt.Print("KubeArmor Annotation Controller TLS certificates already exists ...\n")
+	}
+
+	fmt.Print("KubeArmor Annotation Controller Deployment ...\n")
+	if _, err := c.K8sClientset.AppsV1().Deployments(o.Namespace).Create(context.Background(), deployments.GetAnnotationsControllerDeployment(o.Namespace), metav1.CreateOptions{}); err != nil {
```

Comment:
What happens if we use the annotation controller with our current KubeArmor setup?

I am just wondering if we use `karmor install --image="kubearmor/kubearmor:v2.0"` any other version which doesn't support the annotation controller. Would it cause any issues?

Reply:
It wont have any impact/issues on older version. I run that test and the annotation controller works fine with older versions.